src/App/Pages/ContactPage.elm

```diff
@@ -10,8 +10,18 @@ import Material.Options as Options
 import Material.Button as Button
 
 
-view : Model -> Html Msg
-view model =
+init : ContactForm
+init =
+    { name = ""
+    , email = ""
+    , telephone = ""
+    , subject = ""
+    , message = ""
+    }
+
+
+view : Model -> ContactForm -> Html Msg
```

Comment:
Yeah this is exactly the problem. I defined "ContactMsg" already, but I need another name for the global Msg. To understand you correctly, best practise is to split those pieces up into their own namespace, then naming is easy?